lib/src/Base/Algo/EnclosingSimplexAlgorithm.cxx

```diff
@@ -82,6 +82,7 @@ EnclosingSimplexAlgorithm::EnclosingSimplexAlgorithm(const Sample & vertices, co
     else
       p_implementation_ = new BoundingVolumeHierarchy(vertices, simplices);
   }
+  p_implementation_->setBarycentricCoordinatesEpsilon(ResourceMap::GetAsScalar("EnclosingSimplexAlgorithm-BarycentricCoordinatesEpsilon"));
```

Comment:
isnt this already initialized in the EnclosingSimplexAlgorithmImplementation ctor ?

Reply:
Correct. Fixed.